tests/README.md

```diff
@@ -1,45 +1,53 @@
-## Requirements for running the tests
+# Tests
 
-### Binaries
+## Binaries
 
 The tests run using both the plugin elf and the ethereum elf, both compiled for
-NanoS and X, so 4 binaries are needed. They are expected to be stored in the
-`tests/elfs/` directory, and named as follow:
+Nano (S, S plus, X) and Stax devices, so 8 binaries are needed. The plugin binaries are expected to be stored in the
+`tests/lib_binaries/` directory, and named as follow:
 
-* `tests/elfs/plugin_nanos.elf`
-* `tests/elfs/plugin_nanox.elf`
-* `tests/elfs/ethereum_nanos.elf`
-* `tests/elfs/ethereum_nanox.elf`
+* `tests/lib_binaries/pluginBoilerplate_nanos.elf`
+* `tests/lib_binaries/pluginBoilerplate_nanos2.elf`
+* `tests/lib_binaries/pluginBoilerplate_nanox.elf`
+* `tests/lib_binaries/pluginBoilerplate_stax.elf`
 
-The `tests/build_local_test_elfs.sh` can be used to generate these binaries.
+The ethereum binaries are expected to be stored in the
+`build/` directory, and named as follow:
 
-To use this script, you will need the
-[Ethereum application](https://github.com/LedgerHQ/app-ethereum) to be cloned
-somewhere, and the path of its repository exported in the `APP_ETHEREUM`
-environment variable, or updated in the script, or as an argument of the script.
+* `build/nanos/bin/app.elf`
+* `build/nanos2/bin/app.elf`
+* `build/nanox/bin/app.elf`
+* `build/stax/bin/app.elf`
```

Comment:
It means that the direct compilation output of the app is not used as-is for testing ? We need to copy the boilerplate plugin to the subdir AND copy the eth files to the boilerplate build directory ? Where they will be overwritten if we compile again ?

Reply:
Exactly ;-)
this is one of the improvement we need to do, be able to run easily the test locally

Reply:
Ok, I'll try to solve this by modifying the local ragger conftest of the plugin